src/components/Tool/Appeal/InitialPage/AddAppealForm/AddAppealForm.tsx

```diff
@@ -0,0 +1,661 @@
+              </Grid>
+              <Grid item xs={12} sm={2}>
+                <Box
+                  display="flex"
+                  flexDirection="column"
+                  justifyContent="start"
+                >
+                  <Field
+                    name="adminCost"
+                    type="number"
+                    variant="outlined"
+                    label={t('Admin %')}
+                    size="small"
+                    className={classes.input}
+                    error={errors.adminCost}
+                    helperText={errors.adminCost}
+                    as={TextField}
+                  />
+                </Box>
+              </Grid>
+              <Grid item xs={12} sm={1}>
+                <Box
+                  display="flex"
+                  alignItems="center"
+                  justifyContent="center"
+                  style={{
+                    height: '100%',
+                  }}
+                >
+                  <Icon path={mdiEqual} size={1} />
+                </Box>
+              </Grid>
+              <Grid item xs={12} sm={3}>
+                <Box
+                  display="flex"
+                  flexDirection="column"
+                  justifyContent="start"
+                >
+                  <TextField
+                    data-testid="goalInput"
+                    name="goal"
+                    type="number"
+                    label={t('Goal')}
+                    variant="outlined"
+                    size="small"
+                    className={classes.input}
+                    value={calculateGoal(
+                      initialGoal,
+                      letterCost,
+                      adminCost,
+                    ).toFixed(2)}
+                  />
+                </Box>
+              </Grid>
+            </Grid>
+          </Box>
+          <Alert severity="info">
+            {t(
+              'You can add contacts to your appeal based on their status and/or tags. You can also add additional contacts individually at a later time.',
+            )}
+          </Alert>
+          <Box mt={1} mb={1}>
+            <Typography display="inline">
+              {t('Add contacts with the following status(es):')}
+            </Typography>
+            {!!contactStatuses && (
+              <Typography
+                display="inline"
+                className={classes.selectAll}
+                onClick={() => handleSelectAllStatuses(setFieldValue)}
+                data-testid="contactStatusSelect-selectAll"
+              >
+                {t('select all')}
+              </Typography>
+            )}
+
+            {loadingStatuses && <Skeleton height={40} />}
+
+            {!!contactStatuses && !loadingStatuses && (
+              <Autocomplete
+                multiple
+                autoHighlight
+                data-testid="contactStatusSelect"
+                id="tags-standard"
+                options={contactStatuses.filter(
```

Comment:
It looks like you're filtering out options that are selected. Is there a reason for that?

Reply:
Yes, as they can clear them in the input and make it easier to see what statuses are left

Reply:
Can you try the `filterSelectedOptions` prop instead of filtering manually? https://mui.com/material-ui/react-autocomplete/#multiple-values

Reply:
Amazing! I didn't know that was a thing